src/Cryptol/Parser/Unlit.hs

```diff
@@ -134,6 +137,34 @@ latex = comment []
   isBeginCode l = "\\begin{code}" `Text.isPrefixOf` l
   isEndCode l   = "\\end{code}"   `Text.isPrefixOf` l
 
-
+rst :: [Text] -> [Block]
+rst = comment []
+  where
+  isBeginCode l = case filter (not . Text.null) (Text.splitOn " " l) of
+                    ["..", "code::", "cryptol"] -> True
```

Comment:
I wonder if we should recognize `code-block::` and `sourcecode::` in addition to `code::`? The first two are [officially recognized directives](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-code-block) in Sphinx.

In fact, `code` is technically a [different thing altogether](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-code), although I don't know if strict adherence to Sphinx's rules is a goal for Cryptol.

Reply:
You are quite right, I was reading the docs for `code-block` and I guess auto-corrected it `code` :)